Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 authors = ["Kisio Digital <team.coretools@kisio.com>", "Guillaume Pinot <texitoi@texitoi.eu>"]
 name = "transit_model"
-version = "0.32.3"
+version = "0.32.4"
```

Comment:
No need for a `0.32.4` since `0.32.3` has not yet been released. However, since this PR is changing the behavior of the GTFS export, I'm wondering if it should not be a `0.33.0` instead.

Reply:
@patochectp @woshilapin It's already a `0.33.0` in Cargo.

Reply:
Yes, indeed, because we did merge #726 before waiting for the current PR. Then `0.33.0` it is I guess!